examples/README.md

```diff
@@ -0,0 +1,27 @@
+# Amplitude Python SDK
+
+There are some example apps of using Amplitude Python SDK.
+
+## Usage
+
+### Setup before execution
+
+1. Install Amplitude Python SDK
+   1. `pip install amplitude-analytics`
+2. Replace `YOUR API KEY` in example code with your amplitude project api key.
+
```

Comment:
I think in this top level section we could call out common steps in all examples.

```suggestion
### Setup before execution
All examples require installing the Amplitude Python SDK and setting your API key.

1. Install Amplitude Python SDK `pip install amplitude-analytics`
2. Replace `YOUR API KEY` in example code with your amplitude project api key.

For individual example usage and setup see subproject READMEs.
```